homeassistant/components/plex/media_browser.py

```diff
@@ -94,14 +94,19 @@ def build_item_response(payload):
     if special_folder:
         if media_content_type == "server":
             library_or_section = plex_server.library
+            media_class = MEDIA_CLASS_DIRECTORY
             title = plex_server.friendly_name
         elif media_content_type == "library":
             library_or_section = plex_server.library.sectionByID(media_content_id)
+            media_class = library_or_section.TYPE
```

Comment:
Only our official media class constants are allowed.

Reply:
These match the constants.